app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -385,6 +387,20 @@ open class MainActivity : AppCompatActivity(),
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
+        onBackPressedDispatcher.addCallback(this, object : OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
+                mIsKeyboardOpen = false
+                showActionBarIfNeeded()
+
+                // Disable the callback temporarily to allow the system to handle the back pressed event. This usage
+                // breaks predictive back gesture behavior and should be reviewed before enabling the predictive back
+                // gesture feature.
+                isEnabled = false
+                onBackPressedDispatcher.onBackPressed()
+                isEnabled = true
```

Comment:
1. Praise (❤️): You are becoming a pro at that, aren't you! 😃 
2. Suggestion (💡): Maybe creating a `CompatExtensions.kt` utility class on Aztec, with a `onBackPressedCompat(...)` extension function, copy-pasting it from [here](https://github.com/wordpress-mobile/WordPress-Android/pull/17947/files#diff-b836fc4f03a795e7b9d624220b8ed3d88bbec73d5ecf3848fc55dbee8d5d97a6R16-R30), is the right move for this repo too, just to keep this consistent across every repo, wdyt? 🤔 PS: I know this might seem an overkill as it only applies to this specific and single instance of such `onBackPressedDispatcher` functionality within this repo, so if you want to skip it instead, I would totally understand.